assets/js/edd-checkout-global.js

```diff
@@ -344,15 +344,25 @@ function recalculate_taxes(state) {
 
 	var $edd_cc_address = jQuery('#edd_cc_address');
 
+	var billing_country = $edd_cc_address.find('#billing_country').val(),
+		card_address = $edd_cc_address.find('#card_address').val(),
+		card_address_2 = $edd_cc_address.find('#card_address_2').val(),
+		card_city = $edd_cc_address.find('#card_city').val(),
+		card_state = $edd_cc_address.find('#card_state').val(),
+		card_zip = $edd_cc_address.find('#card_zip').val();
+
 	if( ! state ) {
-		state = $edd_cc_address.find('#card_state').val();
+		state = card_state;
 	}
 
 	var postData = {
 		action: 'edd_recalculate_taxes',
-		billing_country: $edd_cc_address.find('#billing_country').val(),
+		card_address: card_address,
+		card_address_2: card_address_2,
+		card_city: card_city,
+		card_zip: card_zip,
 		state: state,
-		card_zip: $edd_cc_address.find('input[name=card_zip]').val()
+		billing_country: billing_country
```

Comment:
We need to leave `card_zip` in this array of things being posted so that anyone triggering on this JS can still use it, if they are expecting it.

Reply:
Hi @cklosowski , `card_zip` is still in there, just above the `state: state`, item. It's just less noticeable because I moved `card_zip = $edd_cc_address.find('#card_zip').val();` up to the top for consistency's sake (I did the saem for `state = $edd_cc_address.find('#card_state').val();`).

Reply:
@petemolinero Ah, sorry I totally missed that. Thanks! This is fine then.

For future reference, moving things for 'consistency' is better left unless the whole section is refactored as now when trying to find when things were add/removed requires jumping a few different 'blame' commits back. It's not a technical issue, just makes managing change history harder when debugging.

Reply:
@cklosowski Got it! Will remember that in the future!